ams/ams-server/src/main/java/com/netease/arctic/ams/server/ArcticMetaStore.java

```diff
@@ -95,6 +96,80 @@ public static void main(String[] args) throws Throwable {
     }
   }
 
+
+  public static LinkedHashMap getSystemSettingFromYaml() {
+    JSONObject systemConfig = yamlConfig.getJSONObject(ConfigFileProperties.SYSTEM_CONFIG);
+    LinkedHashMap<String, Object> config = new LinkedHashMap<String, Object>();
+
+    if (!systemConfig.containsKey(ArcticMetaStoreConf.THRIFT_BIND_HOST_PREFIX.key())) {
+      throw new RuntimeException("configuration " + ArcticMetaStoreConf.THRIFT_BIND_HOST_PREFIX.key() + " must be set");
+    }
+    InetAddress inetAddress = AmsUtils.getLocalHostExactAddress(
+                    systemConfig.getString(ArcticMetaStoreConf.THRIFT_BIND_HOST_PREFIX.key()));
+    if (inetAddress == null) {
+      throw new RuntimeException("can't find host address start with " +
+              systemConfig.getString(ArcticMetaStoreConf.THRIFT_BIND_HOST_PREFIX.key()));
+    }
+    config.put(ArcticMetaStoreConf.THRIFT_BIND_HOST.key(), inetAddress.getHostAddress());
+
+    String systemThriftPort = System.getProperty(ArcticMetaStoreConf.THRIFT_BIND_PORT.key());
+    if (systemThriftPort == null) {
+      config.put(ArcticMetaStoreConf.THRIFT_BIND_PORT.key(),
+              systemConfig.getInteger(ArcticMetaStoreConf.THRIFT_BIND_PORT.key()));
+    } else {
+      config.put(ArcticMetaStoreConf.THRIFT_BIND_PORT.key(), Integer.parseInt(systemThriftPort));
+    }
+
+    config.put(ArcticMetaStoreConf.HTTP_SERVER_PORT.key(),
+            systemConfig.getInteger(ArcticMetaStoreConf.HTTP_SERVER_PORT.key()));
+    config.put(ArcticMetaStoreConf.OPTIMIZE_CHECK_THREAD_POOL_SIZE.key(),
+            systemConfig.getInteger(ArcticMetaStoreConf.OPTIMIZE_CHECK_THREAD_POOL_SIZE.key()));
+    config.put(ArcticMetaStoreConf.OPTIMIZE_COMMIT_THREAD_POOL_SIZE.key(),
+            systemConfig.getInteger(ArcticMetaStoreConf.OPTIMIZE_COMMIT_THREAD_POOL_SIZE.key()));
+    config.put(ArcticMetaStoreConf.EXPIRE_THREAD_POOL_SIZE.key(),
+            systemConfig.getInteger(ArcticMetaStoreConf.EXPIRE_THREAD_POOL_SIZE.key()));
+    config.put(ArcticMetaStoreConf.ORPHAN_CLEAN_THREAD_POOL_SIZE.key(),
+            systemConfig.getInteger(ArcticMetaStoreConf.ORPHAN_CLEAN_THREAD_POOL_SIZE.key()));
+    config.put(ArcticMetaStoreConf.SYNC_FILE_INFO_CACHE_THREAD_POOL_SIZE.key(),
+            systemConfig.getInteger(ArcticMetaStoreConf.SYNC_FILE_INFO_CACHE_THREAD_POOL_SIZE.key()));
+
+    config.put(ArcticMetaStoreConf.DB_TYPE.key(),
+            systemConfig.getString(ArcticMetaStoreConf.DB_TYPE.key()));
+    config.put(ArcticMetaStoreConf.MYBATIS_CONNECTION_URL.key(),
+            systemConfig.getString(ArcticMetaStoreConf.MYBATIS_CONNECTION_URL.key()));
+    config.put(ArcticMetaStoreConf.MYBATIS_CONNECTION_DRIVER_CLASS_NAME.key(),
+            systemConfig.getString(ArcticMetaStoreConf.MYBATIS_CONNECTION_DRIVER_CLASS_NAME.key()));
+
+    //mysql config
+    if (systemConfig.getString(ArcticMetaStoreConf.DB_TYPE.key()).equalsIgnoreCase("mysql")) {
+      config.put(ArcticMetaStoreConf.MYBATIS_CONNECTION_PASSWORD.key(),
```

Comment:
Can you hide mysql password? Showing it in the front site may be risky

Reply:
The front end web ui had replaced this password with `******`, but I am agree with to hidden this setting in http response too. cc @Aireed